subprojects/core/src/main/java/org/gradle/internal/service/scopes/VirtualFileSystemServices.java

```diff
@@ -264,7 +277,8 @@ public void beforeOutputChange() {
 
                     @Override
                     public void beforeOutputChange(Iterable<String> affectedOutputPaths) {
-                        routingVirtualFileSystem.update(affectedOutputPaths, () -> {});
+                        routingVirtualFileSystem.update(affectedOutputPaths, () -> {
```

Comment:
Why did you add the linebreak here? I think it was better readable before.

Reply:
Accident, I reformatted the whole file and didn't notice that this should have been reverted.

Reply:
Fixed now.